README.md

```diff
@@ -1,3 +1,9 @@
 # Certificate Expiration Exporter
 
+## Try it out locally
+
```

Comment:
Maybe add here the way to create the Google Cloud credentials...or not, just a nit :)